app/services/api/v1x0.rb

```diff
@@ -0,0 +1,42 @@
+require 'services/api/v1x0/catalog'
+require 'services/api/v1x0/catalog/add_to_order'
+require 'services/api/v1x0/catalog/approval_transition'
+require 'services/api/v1x0/catalog/cancel_order'
+require 'services/api/v1x0/catalog/copy_portfolio'
+require 'services/api/v1x0/catalog/copy_portfolio_item'
+require 'services/api/v1x0/catalog/create_approval_request'
+require 'services/api/v1x0/catalog/create_icon'
+require 'services/api/v1x0/catalog/create_request_body_from'
+require 'services/api/v1x0/catalog/create_request_for_applied_inventories'
+require 'services/api/v1x0/catalog/determine_task_relevancy'
+require 'services/api/v1x0/catalog/discard_restore'
+require 'services/api/v1x0/catalog/duplicate_image'
+require 'services/api/v1x0/catalog/import_service_plans'
+require 'services/api/v1x0/catalog/next_name'
+require 'services/api/v1x0/catalog/notify_approval_request'
+require 'services/api/v1x0/catalog/order_item_sanitized_parameters'
+require 'services/api/v1x0/catalog/provider_control_parameters'
+require 'services/api/v1x0/catalog/service_offering'
+require 'services/api/v1x0/catalog/service_plan_compare'
+require 'services/api/v1x0/catalog/service_plan_json'
+require 'services/api/v1x0/catalog/service_plan_reset'
+require 'services/api/v1x0/catalog/service_plans'
+require 'services/api/v1x0/catalog/share_info'
+require 'services/api/v1x0/catalog/share_resource'
+require 'services/api/v1x0/catalog/soft_delete'
+require 'services/api/v1x0/catalog/soft_delete_restore'
+require 'services/api/v1x0/catalog/submit_order'
+require 'services/api/v1x0/catalog/tenant_settings'
+require 'services/api/v1x0/catalog/unshare_resource'
+require 'services/api/v1x0/catalog/update_icon'
+require 'services/api/v1x0/catalog/update_order_item'
+require 'services/api/v1x0/catalog/validate_source'
+require 'services/api/v1x0/catalog/validate_source'
```

Comment:
Might not be important if we use `Dir.glob` but you have `validate_source` listed twice.